src/css/base.scss

```diff
@@ -1671,6 +1671,10 @@ button.anchor {
     max-width: 200px;
 }
 
+.dc__mxw-1000 {
+    max-width: 1000px;
+}
+
 .fw-n {
     font-weight: normal !important;
```

Comment:
we have fw-4 for normal text

Reply:
I didn't use this CSS & removing can impact other files